plugins/logs/plugin.go

```diff
@@ -908,16 +908,11 @@ func uploadChunk(ctx context.Context, client rest.Client, uploadPath string, dat
 
 	defer util.Close(resp)
 
-	switch resp.StatusCode {
-	case http.StatusOK:
-		return nil
-	case http.StatusNotFound:
-		return fmt.Errorf("log upload failed, server replied with not found")
-	case http.StatusUnauthorized:
-		return fmt.Errorf("log upload failed, server replied with not authorized")
-	default:
-		return fmt.Errorf("log upload failed, server replied with HTTP %v", resp.StatusCode)
+	if resp.StatusCode < 200 || resp.StatusCode >= 300 {
+		return fmt.Errorf("log upload failed, server replied with HTTP %v %v", resp.StatusCode, http.StatusText(resp.StatusCode))
```

Comment:
[nit] technically, there could be an extra space at the end here when a status code is returned that doesn't map to a status _text_. But I think practically, it's unlikely, and there's little harm in an extra space.

Reply:
```suggestion
		return errors.New(strings.TrimSpace(
        fmt.Sprintf("log upload failed, server replied with HTTP %v %v", resp.StatusCode, http.StatusText(resp.StatusCode)),
))
```

This would solve the trailing space in case of a status code that is not recognized in net/http, but it makes the code less readable. Let me know which you prefer and I'll change it accordingly.